src/mainboard/pcengines/apu2/mainboard.c

```diff
@@ -250,6 +250,34 @@ static void mainboard_enable(struct device *dev)
 
 static void mainboard_final(void *chip_info)
 {
+	//
+	// AGESA set bit 0, EccExclEn, in register D18F5x240 as 1.
+	// Range was incorrectly enabled to cover all memory
+	// for cases without UMA (no integrated graphics).
+	//
+	// In order to clear EccExclEn DRAM scrubber needs to be
+	// disabled temporarily by setting D18F3x88[DisDramScrub]
```

Comment:
@krystian-hebel that description doesn't explain what is your high level motivation.